src/mp_ui_alerts.cpp

```diff
@@ -50,8 +50,25 @@ bool notif_pref(const std::string& id)
 
 } // end anonymous namespace
 
-// Note: This list must agree with data/gui/.../lobby_sound_options.cfg
-const std::vector<std::string> items {"player_joins", "player_leaves", "private_message", "friend_message", "public_message", "server_message", "ready_for_start", "game_has_begun", "turn_changed"};
+const std::vector<std::string> items {"player_joins", "player_leaves", "private_message", "friend_message", "public_message", "server_message", "ready_for_start", "game_has_begun", "turn_changed", "game_created"};
+
+void game_created(const std::string & scenario, const std::string & name)
+{
+	std::string id = "game_created";
+	if (!lobby_pref(id)) {
+		return ;
+	}
+
+	if (sound_pref(id)) {
+		sound::play_UI_sound(game_config::sounds::game_created);
+	}
+
+	if (notif_pref(id)) {
+		const std::string message = VGETTEXT("A game ($name|, $scenario|) has been created", {{"name", name}, {"scenario", scenario}});
+		desktop::notifications::send(_("Wesnoth"), message, desktop::notifications::OTHER);
+	}
+	
```

Comment:
I'm seeing trailing whitespace on this line in Git, but not in Github's web interface.

Reply:
Also from the original commit. Removed extra line altogether.